src/components/EditPlanter.js

```diff
@@ -91,7 +94,16 @@ const EditPlanter = (props) => {
   }
 
   function handleSelectPlanterImage(img) {
-    handleChange('imageUrl', img);
+    // Planter Manager with Tree Manager role cannot set planter selfie
```

Comment:
This is not correct.
A planter manager should be able to set this image even if they don't have tree permissions.

Reply:
Sorry I wrote wrong comment. It should be "Planter Manager without Tree Manager role cannot set planter selfie". 
So what you mean I should hangle logic when clicking "Save" button instead?

Reply:
No, I mean that we don’t need to check permissions at all.
The title of the issue is a description of the bug, not the desired functionality.